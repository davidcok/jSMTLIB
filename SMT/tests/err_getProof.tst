; tests get-proof without :produce-proofs
(set-logic QF_UF)
(get-proof)
(get-proof xxx)
