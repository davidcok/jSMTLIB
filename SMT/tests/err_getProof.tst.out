success
(get-proof)
^^^^^^^^^^^
(error "The get-proof command is only valid if :produce-proofs has been enabled")
(get-proof xxx)
           ^^^
(error "A get-proof command takes no arguments")
